src/js/rendering/render-genres.js

```diff
@@ -1,31 +1,31 @@
 // ------ GENRES RENDERING ------
 
 // ---- setting fetched array of ids genres to local storage ------
-export const setMovieGenresToStorage = d => {
+export const saveMovieGenresToStorage = d => {
   const movieGenresIds = d.genres;
   localStorage.setItem('movieGenresIdsArray', JSON.stringify(movieGenresIds));
 };
 
 // ---- rendering array of ids genres from local storage to JS ------
-const renderMovieGenres = () => {
+const getMovieGenresFromStorage = () => {
   const movieGenresIdsJSON = localStorage.getItem('movieGenresIdsArray');
   const parsedGenresIDS = JSON.parse(movieGenresIdsJSON);
   return parsedGenresIDS;
 };
 
 // ----- genres and ids array rendered from local storage -----
-const genresIdArray = renderMovieGenres();
+const genresIdArray = getMovieGenresFromStorage();
 console.log(`to jest genresIdArray`);
 console.log(genresIdArray);
 
 // ----- searching movie genre via genre id ------------
```

Comment:
Comment and function name isn't correct! -> This function does replace GenreIds to GenreNames.
Proposition of name: getGenresNamesFromGenreIds or getGenresNamesFromGenreIdsArray

Reply:
"This function does replace GenreIds to GenreNames." - correct spelling: This function replaces GenreIds with GenreNames. 
GenreNames are simply --> Genres. Therefore: GenresIds vs Genres